### PR TITLE
Sprint 14-2 bugfix

### PR DESCRIPTION
**Pull request by BojanSovticEngIT**
Bugfixes:

- https://production.eng.it/jira/browse/KNOWAGE-7104
- https://production.eng.it/jira/browse/KNOWAGE-7077
- [KNOWAGE-7002](https://production.eng.it/jira/browse/KNOWAGE-7002)
- [KNOWAGE-7099](https://production.eng.it/jira/browse/KNOWAGE-7099)

Possible bugfix:
- [KNOWAGE-7102](https://production.eng.it/jira/browse/KNOWAGE-7102)

### CODE REVIEW

File: knowage-vue/src/modules/documentExecution/olap/filterPanel/OlapFilterCard.vue [hunk 1]
```diff
@@ -3,7 +3,13 @@
...
             <span class="p-ml-1"> {{ filter.caption }} </span>
-            <Button icon="fas fa-filter" :class="{ 'olap-active-filter-icon': filterIsActive(filter) }" class="p-button-text p-button-rounded p-button-plain p-ml-auto" @click="openFilterDialog(filter)" />
+            <Button
+                icon="fas fa-filter"
+                :class="{ 'olap-active-filter-icon': filterIsActive(filter) }"
+                class="p-button-text p-button-rounded p-button-plain p-ml-auto"
+                v-tooltip="{ value: getSlicersTooltip(filter), disabled: !filter || !filter.hierarchies[0].slicers || filter.hierarchies[0].slicers.length === 0 }"
+                @click="openFilterDialog(filter)"
+            />
             <!-- TODO Change Request for next sprint: Tooltip for selected filters when hovering on icon and knowage magenta button color if filter is selected -->
```
Comment (n3ils) [inline knowage-vue/src/modules/documentExecution/olap/filterPanel/OlapFilterCard.vue:13]: I see that the toolpit has been already added. Do I need to remove the TODO comment?

Comment (BojanSovticEngIT) [reply knowage-vue/src/modules/documentExecution/olap/filterPanel/OlapFilterCard.vue:13]: I think Darko made that comment. @dbulatovicx32

Comment (dbulatovicx32) [reply knowage-vue/src/modules/documentExecution/olap/filterPanel/OlapFilterCard.vue:13]: You can remove it yes.

Comment (BojanSovticEngIT) [reply knowage-vue/src/modules/documentExecution/olap/filterPanel/OlapFilterCard.vue:13]: You can remove it.

**Review by n3ils — APPROVED** [review]

**Pull request merged**